.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
-reports/
\ No newline at end of file
+reports/
+.idea
```

Comment:
Editor-specific ignores are recommended to be [globally ignored](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) as each developer may use a different editor.